### DISCUSSION TITLE
Vaillant PMS02 and PWM01 detected but no data

### DISCUSSION THREAD

**Discussion by twingo667** [john30/ebusd#545] [Q&A]
Hello,

please see the following:

```
pi@raspberrypi:/etc/ebusd/vaillant $ ebusctl -p 8888
localhost: i
version: ebusd 22.1.v22.1
update check: OK
device: /dev/ttyAMA0, enhanced
access: *
signal: acquired
symbol rate: 23
max symbol rate: 210
min arbitration micros: 3
max arbitration micros: 41
min symbol latency: 10
max symbol latency: 28
reconnects: 0
masters: 6
messages: 823
conditional: 18
poll: 0
update: 10
address 00: master #1
address 01: master #6
address 03: master #11
address 05: slave #1, scanned "MF=Vaillant;ID=VR920;SW=2402;HW=5703"
address 06: slave #6, scanned "MF=Vaillant;ID=PMS02;SW=0209;HW=8402", loaded "vaillant/06.pms.csv"
address 08: slave #11, scanned "MF=Vaillant;ID=BAI00;SW=0603;HW=9102", loaded "vaillant/bai.308523.inc", "vaillant/08.bai.csv"
address 0a: slave, scanned "MF=Vaillant;ID=PMW01;SW=0206;HW=8302", loaded "vaillant/0a.pmw.hwc.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=70000;SW=0510;HW=6403", loaded "vaillant/15.700.csv"
address 26: slave, scanned "MF=Vaillant;ID=VR_71;SW=0104;HW=0503", loaded "vaillant/26.vr_71.csv"
address 31: master #8, ebusd
address 36: slave #8, ebusd
address ec: slave, scanned "MF=Vaillant;ID=70000;SW=0510;HW=6403"
address f7: master #20
address fc: slave #20, scanned "MF=Vaillant;ID=PMW01;SW=0206;HW=8302"

localhost: quit
```

I receive lots of topics regarding VR71, VR920, bai, VR700 but no results of the PWM ans the PMS.

Any idea what I am doing wrong ?

Thanks in advance
Marcus

**Comment by Koky05** [comment]
Try:
ebusctl find -c pwm
ebusctl find -c pmw

If it not return any data, make local copy of configuration files and change priority by adding 1 after first r in line, for example:
`r1,,Ntc1RawData,Ntc1RawData,,,,"0000",,,UIN,,,`

**Comment by twingo667** [comment]
Hi
thanks for your reply.
These are the results:

***@***.***:~ $ ebusctl find -c pwm
ERR: element not found

***@***.***:~ $ ebusctl find -c pmw
ERR: element not found

***@***.***:~ $ ebusctl find -c pms
pms ASIDetected = no data stored
pms BufferControlDiff = no data stored
pms BufferControlInt = no data stored
pms BufferControlPress = no data stored
pms BufferPumpPower = no data stored
pms CountryZone = no data stored
pms CurrentASCStateforASI = no data stored
pms CurrentASCStateforExternals = no data stored
pms CurrentBufferTempDesired = no data stored
pms currenterror = -;-;-;-;-
pms CurrentFlow = no data stored
pms CurrentSolTempDesired = no data stored
pms CurrentSystemMode = no data stored
pms CurrentVPMLoadingMode = no data stored
pms Date = no data stored
pms DateAndTimeIsSet = no data stored
pms errorhistory = no data stored
pms ExternalHcRequest = no data stored
pms ExternalHcTempDesired = no data stored
pms ExternalHwcRequest = no data stored
pms ExternalHwcTempDesired = no data stored
pms FlowSensorFrequency = no data stored
pms ForceStartSolPumpKick = no data stored
pms InternalHcTempDesired = no data stored
pms InternalHwcTempDesired = no data stored
pms MonitorBufferPIDValue = no data stored
pms MonitorMatlabHcCalculation = no data stored
pms MonitorMatlabHwcCalculation = no data stored
pms MonitorMatlabLast5minSolAverage = no data stored
pms MonitorMatlabState = no data stored
pms MonitorOverheatState = no data stored
pms MonitorSolPIDValue = no data stored
pms Ntc1RawData = no data stored
pms Ntc1Sensor = no data stored
pms Ntc2RawData = no data stored
pms Ntc2Sensor = no data stored
pms Ntc3RawData = no data stored
pms Ntc3Sensor = no data stored
pms Ntc4RawData = no data stored
pms Ntc4Sensor = no data stored
pms PowerOnCounter = no data stored
pms RequestForExternalDateTimeSetting = no data stored
pms RTCLowVoltageCounter = no data stored
pms RunTimeBufferPumpHours = no data stored
pms RunTimeBufferPumpMinutes = no data stored
pms RunTimePump1Hours = no data stored
pms RunTimePump1Minutes = no data stored
pms SolControlDiff = no data stored
pms SolControlInt = no data stored
pms SolControlPress = no data stored
pms SolPumpPower = no data stored
pms SolPumpRelay = no data stored
pms Time = no data stored
pms Variante = no data stored
pms YieldCurrentDay = no data stored
pms YieldTotal = no data stored

***@***.***:~ $




________________________________
Von: Koky ***@***.***>
Gesendet: Freitag, 25. Februar 2022 12:37
An: john30/ebusd
Cc: Marcus Schneider (Effizienzbox); Author
Betreff: Re: [john30/ebusd] Vaillant PMS02 and PWM01 detected but no data (Discussion #545)


Try:
ebusctl find -c pwm
ebusctl find -c pmw

If it not return any data, make local copy of configuration files and change priority by adding 1 after first r in line, for example:
r1,,Ntc1RawData,Ntc1RawData,,,,"0000",,,UIN,,,

—
Reply to this email directly, view it on GitHub<https://github.com/john30/ebusd/discussions/545#discussioncomment-2250971>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AEGDUO7MPNEMZY3RWFPOGJ3U45SZBANCNFSM5PKAXGWQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/john30/ebusd/discussions/545#discussioncomment-2250971", "url": "https://github.com/john30/ebusd/discussions/545#discussioncomment-2250971", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by Koky05** [comment]
So it et least found pms device but no data receive jet. Like I wrote I will download local copy of configuration file and change priority like it is descried on top of every file r[1-9] lower number is higher priority to read. Then restart service ebusd with new configuration directed to your local folder.
